ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineTerminalTotalDifficultyUpdated.java

```diff
@@ -24,12 +25,14 @@
 
 public class EngineTerminalTotalDifficultyUpdated extends ExecutionEngineJsonRpcMethod {
 
-  protected EngineTerminalTotalDifficultyUpdated(final Vertx vertx) {
-    super(vertx);
+  protected EngineTerminalTotalDifficultyUpdated(
+      final Vertx vertx, final ProtocolContext protocolContext) {
+    super(vertx, protocolContext);
   }
 
   @Override
   public JsonRpcResponse syncResponse(final JsonRpcRequestContext requestContext) {
+    // TODO: write me
```

Comment:
😄 is it not mandatory for the devnet ?

Reply:
Nope:

https://github.com/ethereum/execution-apis/blob/main/src/engine/interop/specification.md

I think for the f2f we are going to test with a static total difficulty.